doc/ref_model/chapters/chapter04.md

```diff
@@ -262,6 +263,32 @@ Table 4-10 shows performance measurement capabilities.
 
 <p align="center"><b>Table 4-11:</b> Cloud Infrastructure management Resource Management Measurements</p>
 
+<a name="4.1.7"></a>
+### 4.1.7 Acceleration/Offload API Requirements 
+HW Accelerators and Offload functions with abstracted interfaces are preferred and can functionally be interchanged, but their characteristics might vary. It is also likely that the CNFs/VNFs and the Cloud Infrastructure will have certification requirements for the implementations.  An SW implementation of these functions is also often needed to have the same abstracted interfaces for the deployment situations when there are no more HW Accelerator or Offload functions available.
+ 
+For Accelerators and Offload functions with externally exposed differences in their capabilities or management functionality these differences must be clear through the management API either explicit for the differing functions or implicit through the use of a unique APIs.
+ 
+Regardless of exposed or internal different capabilities and characteristics, the operators generally require a choice of multiple implementations also for Accelerators and Offload function realisation, which drive the need for ease of portability in between implementations and vendors.
+ 
+The following table of requirements are derived from the VNF/CNF applications, Cloud Infrastructure and Telco Operators needs to have multiple realisations of HW Acceleration and Offload functions that can also be implemented through SW when no special hardware is available. These requirements should be adopted in Reference Architectures to ensure that the different implementations on the market are as aligned as possible in their interfaces and that HW Acceleration and Offload functions get an efficient ecosystem of accelerators that compete on their technical merits and not through obscure or proprietary interfaces.
+
+**Table 4-12** shows Acceleration/Offload API Capabilities.
+
+<a name="Table4-12"></a>
+
+| Ref       | Acceleration/Offload API Capability | Unit   | Definition/Notes                                                                            |
+|-----------|--------------------------------------------|--------|---------------------------------------------------------------------------------------------|
+| e.api.001 | VNF/CNF usage of Accelerator standard i/f                           | Yes/No | VNF/CNF shall use abstracted standardised interfaces to the Acceleration/Offload functions. This would enable use of HW and SW implementations of the accelerated/offloaded functions from multiple vendors in the Cloud Infrastructure.                        |
+| e.api.002 | Virtualisation Infrastructure SW usage of Accelerator standard i/f  | Yes/No | Virtualisation Infrastructure SW shall use abstracted standardised interfaces to the HW-Acceleration/Offload  function that enables multiple HW and SW implementation in the HW Infrastructure Layer of the accelerated function from multiple vendors  |
```

Comment:
```suggestion
| e.api.002 | Virtualisation Infrastructure SW usage of Accelerator standard i/f  | Yes/No | Virtualisation Infrastructure SW shall use abstracted standardised interfaces to the HW-Acceleration/Offload function. This would enable multiple HW and SW implementations in the HW Infrastructure Layer of the accelerated functions from multiple vendors.  |
```

Reply:
@pgoyal01 Text is corrected in a slightly different way, but hopefully to your approval.